Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     targets: [
         .binaryTarget(
             name: "Mapbox",
-            url: "https://github.com/maplibre/maplibre-gl-native/releases/download/ios-v5.12.2/Mapbox-5.12.2.zip",
-            checksum: "348453495bad198b380e033a6bc6e6134bf71be27116904a25152b93a2d77491")
+            url: "https://github.com/maplibre/maplibre-gl-native/releases/download/ios-v5.13.0/Mapbox-5.13.0.zip",
```

Comment:
I *think* that this can be tested manually, but I don't recall how to do that.  When this hits `main`, then I'll do some testing with an Open Source project that makes uses of this Swift Package.

Thanks for contributing to FOSS4G, @louwers